landingzones/templates/landingzones/project_zones.html

```diff
@@ -21,6 +21,10 @@
         table-layout: fixed;
     }
 
+    .sodar-lz-copy-btn {
+      margin-left: 5px;
```

Comment:
Since we use Bootstrap, you should be able to also do this by adding e.g. `ml-1` or `ml-2` to the classes of the button :) But since we want to pin this to the right end of the column, this is not needed anyway.